emsdk_manifest.json

```diff
@@ -350,6 +350,16 @@
     "activated_cfg": "NODE_JS='%installation_dir%/bin/node%.exe%'",
     "activated_env": "EMSDK_NODE=%installation_dir%/bin/node%.exe%"
   },
+  {
+    "id": "node",
+    "version": "18.13.0",
+    "arch": "aarch64",
+    "bitness": 64,
+    "windows_url": "https://github.com/Windows-on-ARM-Experiments/emsdk/releases/download/v0.1.0-alpha/node-v18.13.0-win-arm64.zip",
```

Comment:
How did you promoted the produced artifact to the release? Somehow form Actions artifacts or manually? I am asking because I want to understand how the version string was added to the ZIP package...

Reply:
It was done manually.

Reply:
Either way, I don't think we should use `v18.13.0` as a version string to prevent confusion since those binaries are built from master.

Reply:
node -v shows exactly this version

Reply:
And what about `%EMDSK_NODE% -V`?

Reply:
you are right, somehow I have got a version from pre-installed node. I will fix it

Reply:
fixed